### DISCUSSION TITLE
Absence of an "I don't know" or "I am unsure" option during selection process

### DISCUSSION THREAD

**Discussion by VeraHeininga** [asreview/asreview#1652] [Q&A]
Dear all,

While using ASReview for selections based on titles and abstracts, we noticed the absence of an "I don't know" or "I am unsure" option. In many instances, our team faces a dilemma in confidently deciding whether to include or exclude a study based on the title and abstract at hand. An option to adopt a neutral stance in such cases—effectively setting the paper aside for reconsideration without immediately influencing the prediction model—would be incredibly beneficial. However, there must be a reason that you decided to not include such an option in the newest version of ASReview. Could you perhaps elaborate a bit on that?

Also, could you provide any insights or advice on how we might navigate this scenario using ASReview? Any guidance or potential solutions you could offer would be greatly appreciated.

Thank you in advance for your attention to this matter and for your support.

Best regards,
Vera Heininga

**Comment by jteijema** [comment]
Hi Vera,

The ASReview team has made the deliberate choice to not include an I Don't Know option into the software, for more than one reason. 

Personally, as a data scientist, I am a bit biased towards a certain explanation, but I know other people in the team would explain it in a different way! Maybe someone else will chime in here too.

The way I see it, is that we made the AI in ASReview to be optimized towards showing you the most important paper as soon as possible. We aren't focusing on labeling correctly; after all, that is your job! Our job is to show you the papers that are most important to be seen by you.  By allowing you to "ignore" certain IDK papers, you bring the model at a disadvantage. IDK papers are both very valuable for the model as a learning element, and for you to see. After all, they might be relevant. We want to both find all relevant papers and all IDK papers before you stop screening.

I can confidently say: don't worry about the model, it can handle doing both. Label them as relevant, as the model needs to find those papers. In an upcoming version, we'll have tags available during the labeling process. You could create a tag with the label "uncertain" for example, and have another screening round especially for all those papers!

I hope that clarifies it a bit!
^Jelle

**Comment by Rensvandeschoot** [comment]
You also might want to read the discussion in https://github.com/asreview/asreview/discussions/1451 and the feature request in https://github.com/asreview/asreview/issues/328
